scala/private/phases/phase_runfiles.bzl

```diff
@@ -38,7 +38,7 @@ def _phase_runfiles_default(ctx, p, _args = struct()):
     return _phase_runfiles(
         ctx,
         _args.transitive_files if hasattr(_args, "transitive_files") else depset(
-            [p.declare_executable, p.java_wrapper] + ctx.files._java_runtime,
```

Comment:
Last I check, the DefaultInfo's executable field is assumed part of the overall runfiles output. So this wasn't needed here.

Reply:
Do you think we can validate it somehow? Do you think tests should have been broken? Maybe it's good enough, not sure

Reply:
IMHO it's good enough because otherwise most of the tests covering binary and test rules would have broken! Additionally, I wouldn't have been able to iterate on #959 which incorporates these changes.

If you want, I can test this against our codebase at Stripe to be extra safe.

Reply:
I think that’s a good idea